src/client/utils/tracking/signing.ts

```diff
@@ -1,9 +1,18 @@
 import { MemberQuery } from 'data/graphql'
 
+export interface AWYWindow extends Window {
+  ReactNativeWebView?: {
+    postMessage: (message: string, targetOrigin: string) => void
```

Comment:
I'm not sure this is still the case but judging from [this github issue comment](https://github.com/react-native-webview/react-native-webview/issues/323#issuecomment-466059732) we may want to omit `targetOrigin` here. Maybe we could reach out to Avy to see if that's true?

Reply:
yes its seems like it should be omitted, good catch!

Reply:
👍